examples/README

```diff
@@ -31,7 +31,7 @@ Below is a brief description of the different directories found here:
   * pylab_examples - The interface to Matplotlib similar to MATLAB.
 
   * tests - Tests used by Matplotlib developers to check functionality.
-    (These tests are still sometimes useful, but mostly developers should
+    (These tests are still sometimes useful, but most developers should
     use the pytest tests which perform automatic image comparison.)
```

Comment:
This whole bullet should just go away as this subdirectory does not exist any longer